fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -197,6 +199,12 @@ public boolean dynamicPartitionExists() {
                 && tableProperty.getDynamicPartitionProperty().isExist();
     }
 
+    public boolean dataSortInfoExists() {
```

Comment:
The `LEXICAL` is also a type of "sort info". So I think the method name or method body should be changed to make it more consistent.

Reply:
done